### ISSUE TITLE
Cannot build IronPython3 for Linux

### ISSUE THREAD

**Issue by datenteiler** [IronLanguages/ironpython3#63]
Hi,

I have Mono 4.2.2 installed on Ubuntu 16.04:

```
$ mono --version
Mono JIT compiler version 4.2.2 (Stable 4.2.2.30/996df3c Mon Feb 15 17:30:30 UTC 2016)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
    TLS:           __thread
    SIGSEGV:       altstack
    Notifications: epoll
    Architecture:  amd64
    Disabled:      none
    Misc:          softdebug 
    LLVM:          supported, not enabled.
    GC:            sgen
```

But I cannot build IronPython3:

```
ironpython3$ make
xbuild Build.proj /t:Build /p:Mono=true;BuildFlavour=Debug
XBuild Engine Version 12.0
Mono, Version 4.2.2.0
Copyright (C) 2005-2013 Various Mono authors

Build started 05.03.2016 12:57:10.
__________________________________________________
Project "/home/christian/git/ironpython3/Build.proj" (Build target(s)):
    Target Bootstrap:
    Skipping target "Bootstrap" because its outputs are up-to-date.
    Target _MakePackagesConfig:
    Skipping target "_MakePackagesConfig" because its outputs are up-to-date.
    Target RestoreReferences:
    Skipping target "RestoreReferences" because its outputs are up-to-date.
    Target Build:
        Project "/home/christian/git/ironpython3/IronPython.sln" (Build target(s)):
            Target ValidateSolutionConfiguration:
                Building solution configuration "v2Debug|Any CPU".
            Target Build:
                Project "/home/christian/git/ironpython3/Src/IronPython/IronPython.csproj" (default target(s)):
                    Target Bootstrap:
                    Skipping target "Bootstrap" because its outputs are up-to-date.
                    Target GenerateBuildInfo:
                    Skipping target "GenerateBuildInfo" because its outputs are up-to-date.
                    Target GenerateCurrentVersion:
                    Skipping target "GenerateCurrentVersion" because its outputs are up-to-date.
                    Target PrepareForBuild:
                        Configuration: v2Debug Platform: AnyCPU
                    Target GenerateSatelliteAssemblies:
                    No input files were specified for target GenerateSatelliteAssemblies, skipping.
                Done building project "/home/christian/git/ironpython3/Src/IronPython/IronPython.csproj".-- FAILED
                Project "/home/christian/git/ironpython3/Src/IronPythonTestConsole/IronPythonTestConsole.csproj" (default target(s)):
                    Target PrepareForBuild:
                        Configuration: v2Debug Platform: AnyCPU
                    Target GenerateSatelliteAssemblies:
                    No input files were specified for target GenerateSatelliteAssemblies, skipping.
                Done building project "/home/christian/git/ironpython3/Src/IronPythonTestConsole/IronPythonTestConsole.csproj".-- FAILED
            Task "MSBuild" execution -- FAILED
            Done building target "Build" in project "/home/christian/git/ironpython3/IronPython.sln".-- FAILED
        Done building project "/home/christian/git/ironpython3/IronPython.sln".-- FAILED
    Task "MSBuild" execution -- FAILED
    Done building target "Build" in project "/home/christian/git/ironpython3/Build.proj".-- FAILED
Done building project "/home/christian/git/ironpython3/Build.proj".-- FAILED

Build FAILED.
     0 Warning(s)
     0 Error(s)

Time Elapsed 00:00:01.9693310

```

And xbuild doesn't work too:

```
XBuild Engine Version 12.0
Mono, Version 4.2.2.0
Copyright (C) 2005-2013 Various Mono authors

Build started 05.03.2016 12:58:32.
__________________________________________________
Project "/home/christian/git/ironpython3/IronPython.sln" (default target(s)):
    Target ValidateSolutionConfiguration:
        Building solution configuration "AndroidDebug|Any CPU".
    Target Build:
        Project "/home/christian/git/ironpython3/Src/IronPython/IronPython.csproj" (default target(s)):
            Target Bootstrap:
            Skipping target "Bootstrap" because its outputs are up-to-date.
            Target GenerateBuildInfo:
            Skipping target "GenerateBuildInfo" because its outputs are up-to-date.
            Target GenerateCurrentVersion:
            Skipping target "GenerateCurrentVersion" because its outputs are up-to-date.
            Target PrepareForBuild:
                Configuration: AndroidDebug Platform: AnyCPU
            Target GetReferenceAssemblyPaths:
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Unable to find framework corresponding to the target framework moniker 'MonoAndroid,Version=v2.2'. Framework assembly references will be resolved from the GAC, which might not be the intended behavior.
            Target ResolveAssemblyReferences:
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/../Util/References/MonoAndroid/1.0/mscorlib.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/mscorlib' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/mscorlib.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/mscorlib.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/../Util/References/MonoAndroid/1.0/System.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e', but an assembly with the specific version not found.
                For searchpath {RawFileName}
                Considered 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/../Util/References/MonoAndroid/1.0/System.Core.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e', but an assembly with the specific version not found.
                For searchpath {RawFileName}
                Considered 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.Core' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.Core.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.Core.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Scripting.Core' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/Build/../Util/References/DynamicLanguageRuntime.1.2.0-alpha1/lib/android/Microsoft.Scripting.Core.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered Microsoft.Scripting.Core, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered Microsoft.Scripting.Core, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'Microsoft.Scripting.Core' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.Core' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.Core.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.Core.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Scripting' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/Build/../Util/References/DynamicLanguageRuntime.1.2.0-alpha1/lib/android/Microsoft.Scripting.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered Microsoft.Scripting, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered Microsoft.Scripting, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'Microsoft.Scripting' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Dynamic' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/Build/../Util/References/DynamicLanguageRuntime.1.2.0-alpha1/lib/android/Microsoft.Dynamic.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered Microsoft.Dynamic, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered Microsoft.Dynamic, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'Microsoft.Dynamic' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Dynamic' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Dynamic.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Dynamic.dll' as a file, but the file does not exist
            Target GenerateSatelliteAssemblies:
            No input files were specified for target GenerateSatelliteAssemblies, skipping.
            Target GenerateTargetFrameworkMonikerAttribute:
            Skipping target "GenerateTargetFrameworkMonikerAttribute" because its outputs are up-to-date.
        Done building project "/home/christian/git/ironpython3/Src/IronPython/IronPython.csproj".-- FAILED
        Project "/home/christian/git/ironpython3/Src/IronPythonTestConsole/IronPythonTestConsole.csproj" (default target(s)):
            Target PrepareForBuild:
                Configuration: AndroidDebug Platform: AnyCPU
            Target GetReferenceAssemblyPaths:
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Unable to find framework corresponding to the target framework moniker 'MonoAndroid,Version=v2.2'. Framework assembly references will be resolved from the GAC, which might not be the intended behavior.
            Target ResolveAssemblyReferences:
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/../Util/References/MonoAndroid/1.0/mscorlib.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/mscorlib' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/mscorlib.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/mscorlib.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/../Util/References/MonoAndroid/1.0/System.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e', but an assembly with the specific version not found.
                For searchpath {RawFileName}
                Considered 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/../Util/References/MonoAndroid/1.0/System.Core.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e', but an assembly with the specific version not found.
                For searchpath {RawFileName}
                Considered 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.Core' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.Core.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/System.Core.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Scripting.Core' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/Build/../Util/References/DynamicLanguageRuntime.1.2.0-alpha1/lib/android/Microsoft.Scripting.Core.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered Microsoft.Scripting.Core, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered Microsoft.Scripting.Core, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'Microsoft.Scripting.Core' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.Core' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.Core.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.Core.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Scripting' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/Build/../Util/References/DynamicLanguageRuntime.1.2.0-alpha1/lib/android/Microsoft.Scripting.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered Microsoft.Scripting, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered Microsoft.Scripting, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'Microsoft.Scripting' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Scripting.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Dynamic' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/Build/../Util/References/DynamicLanguageRuntime.1.2.0-alpha1/lib/android/Microsoft.Dynamic.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered Microsoft.Dynamic, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered Microsoft.Dynamic, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'Microsoft.Dynamic' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Dynamic' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Dynamic.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/Microsoft.Dynamic.dll' as a file, but the file does not exist
/usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'nunitlite' not resolved
                For searchpath {CandidateAssemblyFiles}
                Warning: {CandidateAssemblyFiles} not supported currently
                For searchpath {HintPathFromItem}
                Considered /home/christian/git/ironpython3/Build/../Util/References/NUnitLite.1.0.0/lib/android/nunitlite.dll, but it does not exist.
                For searchpath {TargetFrameworkDirectory}
                For searchpath {PkgConfig}
                Considered nunitlite, but could not find in any pkg-config files.
                For searchpath {GAC}
                Considered nunitlite, but could not find in the GAC.
                For searchpath {RawFileName}
                Considered 'nunitlite' as a file, but the file does not exist
                For searchpath /home/christian/git/ironpython3/Build/../bin/AndroidDebug/
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/nunitlite' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/nunitlite.exe' as a file, but the file does not exist
                Considered '/home/christian/git/ironpython3/bin/AndroidDebug/nunitlite.dll' as a file, but the file does not exist
            Target GenerateSatelliteAssemblies:
            No input files were specified for target GenerateSatelliteAssemblies, skipping.
            Target GenerateTargetFrameworkMonikerAttribute:
            Skipping target "GenerateTargetFrameworkMonikerAttribute" because its outputs are up-to-date.
        Done building project "/home/christian/git/ironpython3/Src/IronPythonTestConsole/IronPythonTestConsole.csproj".-- FAILED
    Task "MSBuild" execution -- FAILED
    Done building target "Build" in project "/home/christian/git/ironpython3/IronPython.sln".-- FAILED
Done building project "/home/christian/git/ironpython3/IronPython.sln".-- FAILED

Build FAILED.

Warnings:

/home/christian/git/ironpython3/IronPython.sln (default targets) ->
(Build target) ->
/home/christian/git/ironpython3/Src/IronPython/IronPython.csproj (default targets) ->
/usr/lib/mono/4.5/Microsoft.Common.targets (GetReferenceAssemblyPaths target) ->

    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Unable to find framework corresponding to the target framework moniker 'MonoAndroid,Version=v2.2'. Framework assembly references will be resolved from the GAC, which might not be the intended behavior.

/home/christian/git/ironpython3/IronPython.sln (default targets) ->
(Build target) ->
/home/christian/git/ironpython3/Src/IronPython/IronPython.csproj (default targets) ->
/usr/lib/mono/4.5/Microsoft.Common.targets (ResolveAssemblyReferences target) ->

    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Scripting.Core' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Scripting' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Dynamic' not resolved

/home/christian/git/ironpython3/IronPython.sln (default targets) ->
(Build target) ->
/home/christian/git/ironpython3/Src/IronPythonTestConsole/IronPythonTestConsole.csproj (default targets) ->
/usr/lib/mono/4.5/Microsoft.Common.targets (GetReferenceAssemblyPaths target) ->

    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Unable to find framework corresponding to the target framework moniker 'MonoAndroid,Version=v2.2'. Framework assembly references will be resolved from the GAC, which might not be the intended behavior.

/home/christian/git/ironpython3/IronPython.sln (default targets) ->
(Build target) ->
/home/christian/git/ironpython3/Src/IronPythonTestConsole/IronPythonTestConsole.csproj (default targets) ->
/usr/lib/mono/4.5/Microsoft.Common.targets (ResolveAssemblyReferences target) ->

    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Scripting.Core' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Scripting' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'Microsoft.Dynamic' not resolved
    /usr/lib/mono/4.5/Microsoft.Common.targets:  warning : Reference 'nunitlite' not resolved

     15 Warning(s)
     0 Error(s)

Time Elapsed 00:00:01.8816460

```

What do I miss?

Thanks in advance.

**Comment by slide** [comment]
This looks like it is happening because the Csc tool is being overridden. I am working on a fix.

**Comment by slide** [comment]
I have different errors now it looks like.

**Comment by ashes999** [comment]
I have Mono 4.4.2 (not 4.2.2) installed on Ubuntu 14.04 and I get a different set of compiler errors:

```
/ironpython3/Src/IronPython.Wpf/IronPython.Wpf.csproj (default targets) ->
/usr/lib/mono/4.5/Microsoft.CSharp.targets (CoreCompile target) ->

	wpf.cs(36,76): error CS7069: Reference to type `Microsoft.Scripting.IValueEquality' claims it is defined assembly `Microsoft.Dynamic, Version=1.2.0.0, Culture=neutral, PublicKeyToken=7f709c5b713576e1', but it could not be found
	wpf.cs(36,76): error CS7069: Reference to type `System.Collections.IStructuralEquatable' claims it is defined assembly `Microsoft.Dynamic, Version=1.2.0.0, Culture=neutral, PublicKeyToken=7f709c5b713576e1', but it could not be found

ironpython3/Src/IronPython.SQLite/IronPython.SQLite.csproj (default targets) ->
/usr/lib/mono/4.5/Microsoft.CSharp.targets (CoreCompile target) ->

	c#sqlite/Delegates.cs(221,19): error CS0305: Using the generic type `System.Action<T1,T2,T3,T4>' requires `4' type argument(s)
	c#sqlite/Delegates.cs(247,19): error CS0305: Using the generic type `System.Func<T1,T2,T3,T4,TResult>' requires `5' type argument(s)
	c#sqlite/Delegates.cs(255,19): error CS0305: Using the generic type `System.Func<T1,T2,T3,T4,TResult>' requires `5' type argument(s)
```

I also get some warnings.

Maybe it's a stretch to suggest that this has been broken since March (~6 months), but anyway, would it be too much work for someone to generate a NuGet package for this? Even if it's tagged as v0.0.1-alpha.

**Comment by slide** [comment]
Are you looking for IronPython 3 specifically? It's still a ways out from being feature complete for Python 3.5 compatibility. If you just want IronPython, I suggest you use "main" which is Python 2.7 compatible.

**Comment by ashes999** [comment]
Yes, I'm looking for IronPython 3 specifically. I'm using 2.7 because I couldn't get IronPython 3 to build.

TLDR: this is not for production code, I'm just messing around with embedding IronPython in MonoGame so I can write my game code in Python; see [this repository](https://github.com/ashes999/infinite-arpg) if interested in details.

**Comment by slide** [comment]
We now have IronPython3 up on Travis CI for automated builds, and it currently builds (but does not pass tests) on Linux. I'm closing this out.
